eslint.config.js

```diff
@@ -18,7 +18,7 @@ export default [
     }
   },
   {
-    files: ['web/pages/**/*', 'web/ui/**/*', 'web/main/**/*', 'core/test/**/*'],
+    files: ['web/pages/**/*', 'web/ui/**/*', 'web/main/**/*', 'core/test/**/*', 'scripts/*'],
```

Comment:
We disable this rule only for code running browser.

We can disable it for the whole project or make an allow-list of experimental API for Node.js related code like scripts.

Reply:
Would that work?

```js
{
    files: ['scripts/**.ts'],
    rules: {
      'n/no-unsupported-features/node-builtins': ['error', {
          ignores: ['fs.globSync']
      }]
    }
  }
```

Reply:
Done